### PR TITLE
David fix states

### PR DESCRIPTION
**Pull request by DavidMeagher1**

### CODE REVIEW

File: addons/States/States/State.gd [hunk 1]
```diff
@@ -17,8 +17,8 @@ signal transition_requested(state: State)
 @export var is_enabled: bool = true : set = _set_enabled, get = _get_enabled
 
 func _set_enabled(value: bool):
-	is_enabled = value
 	process_mode = PROCESS_MODE_PAUSABLE if value else PROCESS_MODE_DISABLED
+	is_enabled = value
```
Comment (LeonardMeagher2) [inline addons/States/States/State.gd:21]: Why did we do this?

**Review by LeonardMeagher2 — APPROVED** [review]
Only one question, the rest looks good

File: addons/States/States/State.gd (hunk 1 above)
Comment (DavidMeagher1) [reply addons/States/States/State.gd:21]: I was checking if it was an order issue when disabling states, it didn't seem to change anything so I left it in.

Comment (LeonardMeagher2) [reply addons/States/States/State.gd:21]: I think this would trigger the notification for paused, before setting is_enabled now

Comment (LeonardMeagher2) [inline addons/States/States/State.gd:21]:
```suggestion
	is_enabled = value
	process_mode = PROCESS_MODE_PAUSABLE if value else PROCESS_MODE_DISABLED
```

**Pull request merged**